trieste/models/gpflux/models.py

```diff
@@ -143,10 +144,17 @@ def optimize(self, dataset: Dataset) -> None:
         Optimize the model with the specified `dataset`.
         :param dataset: The data with which to optimize the `model`.
         """
-        self.model_keras.fit(
-            {"inputs": dataset.query_points, "targets": dataset.observations}, **self._fit_args
+        fit_args = self._fit_args
+        if fit_args is not None and "epochs" in fit_args:
+            fit_args["epochs"] = fit_args["epochs"] + self.absolute_epochs
+
+        hist = self.model_keras.fit(
+            {"inputs": dataset.query_points, "targets": dataset.observations},
+            **fit_args,
+            initial_epoch=self.absolute_epochs,
         )
 
+        self.absolute_epochs = self.absolute_epochs + len(hist.history["loss"])
```

Comment:
maybe add a comment explaining what's happening here?

Reply:
this would make it a default behaviour, right?
I don't think we should do that, we should leave it as optional through an argument at init

Reply:
why not? if anything, it improves performance...

Reply:
its a behaviour I would not expect and there is now way to switch it off, right?
we can leave it as a default but allow for turning it off, how does that sound?

Reply:
sounds good

Reply:
The only way I see to do that is with an extra boolean argument at init(). Is this OK?

Reply:
@sebastianober any opinion?

Reply:
I think another argument in the init is fine, we don't have that many arguments in any case